### ISSUE TITLE
Feedback Request: Opdracht-3-AWS-11 ELB & Auto Scaling

### ISSUE THREAD

**Issue by Acron0815** [TechGrounds-Cloud8/cloud8-Acron0815#38]
**Feedback Request [Opdracht-3-AWS-11 ELB & Auto Scaling]**
Permalink to file: [LINK](https://github.com/TechGrounds-Cloud8/cloud8-Acron0815/blob/main/Week%205/Opdracht-3-AWS-11%20ELB%20%26%20Auto%20Scaling.md)

Template for peers for review: 
```
**Summary**
- Does the summary describe the subject matter fully?
- Is the summary written in own words?
- Is there crucial information missing in the summary?
- Is the description clear enough?

**Key Terminology**
- Are there missing key terms?
- Are the key terms explained well?

**Sources**
- Are the sources mentioned?
- Are the mentioned sources correct?

**Excercise**
(Where applicable)
- Is the result correct?
- Are the experienced problems and solutions described well?
```

Share the link to this issue with your peers.

**Comment by yismailmo** [comment]
Yes, Agree with ben here about terminating of  resources and it's also a good practice to look into cost management and also terminate all resources which are not used (especially after each exercise that's done).

**Comment by Acron0815** [comment]
Thanks for the comments, I've added and changed various points.
I should also investigate and check a few things as soon as possible.

PS I will finish all resources!

**Comment by Acron0815** [comment]
I have looked at this exercise and improved it where necessary.
I have to say, thanks to your constructive cooperation, it now looks better and clearer.

**Comment by Acron0815** [comment]
Oh, I almost forgot, the resources are closed now too.
